signac/contrib/filesystems.py

```diff
@@ -27,12 +28,19 @@
 GRIDFS_LARGE_FILE_WARNING_THRSHLD = int(1e9)  # 1GB
 FILESYSTEM_REGISTRY = dict()
 
+# this is here because of issues importing the same variable in
+# signac/__init__.py from the top level namespace
+__version__ = '1.2.0'
 
+
+@deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+            details="No references to this function outside the module.")
 def _register_fs_class(fs):
```

Comment:
Functions and classes like these are used by public functions and classes, so we cannot just remove them, especially since marking things @deprecated just generates warnings, not errors.

Reply:
In this case only the related public classes and functions should be deprecated, not the internal functions.